aioleakybucket/test_static.py

```diff
@@ -0,0 +1,42 @@
+import static
+
+zones = {
+    'login': {
+        'size': 1000,
+        'rate': 1,
+    }
+}
+
+resources = {
+    '/login' : {
+        'zone': 'login',
+        'burst': 2,
+        'delay': 0,
+    }
+}
+
+
+
+def try_requests(limiter, requests):
+    for i, request in enumerate(requests, 1):
```

Comment:
for static version we need to operate on list of tuples (or `pandas.DataFrame` and have list or dataframe with results back)